app/models/concerns/visible.rb

```diff
@@ -0,0 +1,19 @@
+module Visible
+  extend ActiveSupport::Concern
```

Comment:
@sansuaza how the concerns operate?

Reply:
if a functionality is needed in diferents models, this functionlity can be given to the concern, mainly methods class